portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("message");
+    String nameValue = request.getParameter("name");
+    String emailValue = request.getParameter("email");
+    String subjectValue = request.getParameter("subject");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("Name: " + nameValue + "\n" + "Email: " + emailValue 
```

Comment:
In industrial usually a more standard logging is preferred such as: https://www.tutorialspoint.com/servlets/servlets-debugging.htm. But in your case System print definitely works :)

Also string template makes code more readable
```
String.format("Name: %s\nEmail: %s\nSubject: %s\nMessage: %s", nameValue, emailValue, subjectValue, textValue);
```